handbook/engineering/distribution/goals.md

```diff
@@ -67,6 +67,16 @@ Creating a new release for our deployments is currently a semi-automated process
   - Document project and folder usage guidelines.
   - Set spending limits for dynamic environments.
 
+### Improve Sourcegraph monitoring developer experience
+
+- **Owner**: Gonza, Robert
```

Comment:
@pecigonzalo not sure who to add as owners here, saw that we generally seem to have 2 owners so I just added the two of us

Reply:
Its generally the one spearheading the project, I think it would make sense for it to be you and @davejrt as I can only sporadically work on project unfortunately.